python-sdk/tests_integration/databricks_tests/test_delta.py

```diff
@@ -131,7 +131,6 @@ def test_create_table_from_select_statement(database_table_fixture):
 
     df = database.hook.get_pandas_df(f"SELECT * FROM {target_table.name}")
     assert len(df) == 26
-    assert df.NAME[0] == "Dancer"
```

Comment:
Does it make sense to check if the value is in any field - instead of always the first one?

Reply:
make sense, checking if value is in any

Reply:
I'm a bit confused as to why this would be flaky? Seems like the table should be loaded the same way it was in the file

Reply:
From the test looks like we are loading a folder. I checked the folder in s3 and it has two csv files, one file first row has the value "Dancer" for the field "Name" and the other file has the value "Stanley" for the field "Name".

error we are getting in CI
```
FAILED tests_integration/databricks_tests/test_delta.py::test_create_table_from_select_statement[delta] - AssertionError: assert 'Stanley' == 'Dancer'
  - Dancer
  + Stanley
```
https://github.com/astronomer/astro-sdk/actions/runs/3829415846/jobs/6516092626
https://github.com/astronomer/astro-sdk/actions/runs/3831362066/jobs/6520398696

Reply:
@dimberman ^^

Reply:
Ohhh ok so I guess loading to databricks isn't deterministic. (I guess that it makes sense since it's using a distributed spark cluster to load)